README.md

```diff
@@ -416,10 +409,17 @@ The above walkthrough follows the actual implementation process used to produce
 
 ## Manage the operator using the Operator Lifecycle Manager
 
+> NOTE: This section of the Getting Started Guide is out-of-date.
+> We're working on some improvements to Operator SDK to streamline
+> the experience of using OLM. For further information see, for example, the [proposal][sdk-integration-with-olm-doc]. 
+> In the meantime, you might find the following documentation helpful: 
```

Comment:
```suggestion
```

Reply:
I would prefer to keep this part because if we remove it and just keep the note then users may not check the following steps since they will assume that they will not work.